scripts/acceptance.sh

```diff
@@ -5,6 +5,6 @@ set -e -x
 mocha \
   --retries 5 \
   --require test.config.js \
-  -t 30000 \
+  -t 60000 \
   $@ \
   dotenv_config_path=.env.test
```

Comment:
@yuki24 @damassi There’s [a ‘acceptance test timeout’ setting](https://github.com/artsy/force/blob/d861cddc9cb009921c1e232f39d7793b05b9d761/.env.test#L28) in the `.env.test` file, but that’s not being used because the shell is interpreting this option as a test file path. Should this have been `env dotenv_config_path=.env.test mocha […]`?